### PR TITLE
Feature/p2p patch

### PR DESCRIPTION
**Pull request by dhanuwat**
P2P feature from customer (without QuickExchange, CanCan, Abilities).

### CODE REVIEW

File: web/src/components/CustomInput/CustomInput.pcss [hunk 1]
```diff
@@ -16,7 +16,6 @@
 
   input,
   input:focus {
-    background: var(--input-background-color) !important;
```
Comment (josadcha) [inline web/src/components/CustomInput/CustomInput.pcss:19]: please put input-background-color back

File: web/src/components/OrderInput/index.tsx [hunk 2]
```diff
@@ -71,13 +87,16 @@ export const OrderInput: React.FunctionComponent<OrderInputProps> = React.memo((
...
+                    label={(value || labelVisible) && label ? label : ''}
+                    labelVisible={labelVisible}
+                    defaultLabel={(value || labelVisible) && label ? label : ''}
                     onKeyPress={onKeyPress}
                     handleFocusInput={() => handleFocusInput(props.label)}
+                    autoFocus={autoFocus}
                 />
             </fieldset>
             <div className={cryptoIconClass}>
+                { showAllButton && <span className="cr-order-input__crypto-icon-all" onClick={handleClickAllButton}>ALL</span> }
```
Comment (josadcha) [inline web/src/components/OrderInput/index.tsx:99]: ALL should be added to translation files

File: web/src/constants/index.ts [hunk 3]
```diff
@@ -1,27 +1,32 @@
...
     const routes = [
         ['page.header.navbar.trade', '/trading/', `trade${isLight ? 'Light' : ''}`],
-        ['page.header.navbar.quick.exchange', '/quick-exchange', `quick_exchange${isLight ? 'Light' : ''}`],
+        CanCan.checkAbilityByAction('read', 'QuickExchange', abilities) && ['page.header.navbar.quick.exchange', '/quick-exchange', `quick_exchange${isLight ? 'Light' : ''}`],
         ['page.header.navbar.wallets', '/wallets', `wallets${isLight ? 'Light' : ''}`],
         ['page.header.navbar.openOrders', '/orders', `orders${isLight ? 'Light' : ''}`],
         ['page.header.navbar.history', '/history', `history${isLight ? 'Light' : ''}`],
-        ['page.header.navbar.api', '/docs', `api${isLight ? 'Light' : ''}`],
+        CanCan.checkAbilityByAction('read', 'P2P', abilities) && ['page.header.navbar.p2p', '/p2p', `p2p${isLight ? 'Light' : ''}`],
+        ['page.header.navbar.api', '/api', `api${isLight ? 'Light' : ''}`],
```
Comment (josadcha) [inline web/src/constants/index.ts:15]: check please if /api exists on Baseapp

**Review by josadcha — CHANGES_REQUESTED** [review]

File: web/src/components/CustomInput/CustomInput.pcss (hunk 1 above)
Comment (dhanuwat) [reply web/src/components/CustomInput/CustomInput.pcss]: Fixed, thanks.

File: web/src/components/OrderInput/index.tsx (hunk 2 above)
Comment (dhanuwat) [reply web/src/components/OrderInput/index.tsx]: Fixed, thanks.

File: web/src/constants/index.ts (hunk 3 above)
Comment (dhanuwat) [reply web/src/constants/index.ts]: Changed to /docs as /api is not exists, thanks.

**Comment by josadcha** [comment]
duplicate of https://github.com/openware/baseapp/pull/576

**Pull request closed without merging**